### PR TITLE
Fix the updating of the protocol wallet balances on the simulator

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

File: bancor_research/bancor_simulator/v3/spec/actions.py [hunk 1]
```diff
@@ -342,9 +342,8 @@ def process_withdrawal(
             bnbnt_renounced = bnbnt_rate * bnt_renounced
 
             state.decrease_pooltoken_balance("bnbnt", bnbnt_renounced)
-            state.decrease_protocol_wallet_balance("bnbnt", bnbnt_renounced)
+            state.decrease_protocol_wallet_balance("bnt", bnbnt_renounced)
             state.set_tkn_trading_liquidity(tkn_name, updated_tkn_liquidity)
-            state.decrease_protocol_wallet_balance(f"bn{tkn_name}", pool_token_amt)
```
Comment (mikewcasale) [inline bancor_research/bancor_simulator/v3/spec/actions.py:347]: this should be changed to `state.decrease_pooltoken_balance(tkn_name, pool_token_amt)`

File: bancor_research/bancor_simulator/v3/spec/actions.py [hunk 2]
```diff
@@ -342,9 +342,8 @@ def process_withdrawal(
             bnbnt_renounced = bnbnt_rate * bnt_renounced
 
             state.decrease_pooltoken_balance("bnbnt", bnbnt_renounced)
```
Comment (mikewcasale) [inline bancor_research/bancor_simulator/v3/spec/actions.py:344]: this should be changed to `state.decrease_pooltoken_balance("bnt", bnbnt_renounced)`

File: bancor_research/bancor_simulator/v3/spec/actions.py (hunk 1 above)
Comment (barakman) [reply bancor_research/bancor_simulator/v3/spec/actions.py:347]: I tried that, and got a negative balance on bnTKN.

Comment (barakman) [reply bancor_research/bancor_simulator/v3/spec/actions.py:347]: Fixed

File: bancor_research/bancor_simulator/v3/spec/actions.py (hunk 2 above)
Comment (barakman) [reply bancor_research/bancor_simulator/v3/spec/actions.py:344]: Fixed

**Pull request merged**